### PR TITLE
stats: Log packets/bytes per second in stats - V2

### PR DESCRIPTION
**Pull request by I3run0**
This commit aims to add packets/bytes per second to Suricata's statistics log. To achieve this, a new type of qualifier was created and two new fields were added to the StatsRecord struct.

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: [6410](https://redmine.openinfosecfoundation.org/issues/6410)

Previous PR: #9711

Suricata-verify tests: https://github.com/OISF/suricata-verify/pull/1453

Describe changes:
- Fix StatsRegisterTmdCounter function description 
- Add tests to Suricata-verify

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1453

### CODE REVIEW

**Comment by jufajardini** [comment]
Updated PR description to refer to SV PR in format recognized by our CI checks, and re-triggered CI build runs.

**Review by jufajardini — CHANGES_REQUESTED** [review]
Hi there, this is a nit review only, as I'll wait for Lukas to follow up on this previous review.

Focusing on the commit message only:
- nit: typo - `StatsRecord` struct, right?
- there's no need to mention "Suricata" in the commit title, as this repo is Suricata's repo. Can you reword it to reduce redundancy, taking that into account? Thanks!

File: src/counters.c [hunk 1]
```diff
@@ -789,6 +792,10 @@ static int StatsOutput(ThreadVars *tv)
                         r->value = (uint64_t)(e->value / e->updates);
                     }
                     break;
+                case STATS_TYPE_TIMED:
+                    r->value = r->counter_value - r->counter_pvalue;
+                    r->value = (uint64_t)(r->value / stats_tts);
```
[the remark this answers is not in the document]
Comment (I3run0) [reply src/counters.c:797]: No! I haven't checked it!  

Thank you for the comment I will fix this error!

**Comment by catenacyber** [comment]
Closing as stale and changes requested
